samples/BenchmarkDotNet.Samples/JIT/Jit_Inlining.cs

```diff
@@ -5,6 +5,9 @@ namespace BenchmarkDotNet.Samples.JIT
 {
     // See http://en.wikipedia.org/wiki/Inline_expansion
     // See http://aakinshin.net/en/blog/dotnet/inlining-and-starg/
+#if !CORE
+    [Diagnostics.Windows.Configs.InliningDiagnoserConfig]
```

Comment:
Yes and no ;)

In general these attributes have to have dependency to our Diagnosers, and Diagnosers are in the `BenchmarkDotNet.Diagnostics.Windows` project, which supports only NET 4.5+, no .NET Core support.

But we could workaround that by adding the class with same name, but without implementation to our main project. Then all classic projects that have dependency to our Diagnostics package would use the real one, and the .NET Core ones would get empty class.  By using the Obsolete attribute we can give them warning.

```cs
#if CORE
using System;
using BenchmarkDotNet.Configs;

namespace BenchmarkDotNet.Diagnostics.Windows.Configs
{
    [Obsolete("InliningDiagnoser is not supported for .NET Core yet")]
    public class InliningDiagnoserAttribute : Attribute, IConfigSource
    {
        public IConfig Config { get; }

        public InliningDiagnoserAttribute() { }
    }
}
#endif
```


What do you think? Personally I am afraid that this could cause some problems in the future and we would gain only simplier sample.